modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -5,250 +5,249 @@ endif::[]
+//3
+. When the Batch Job component receives a message from an upstream processor in the flow, the xref:batch-phases.adoc#phase_load_dispatch[Load and Dispatch phase] begins. In this phase, the component prepares the input for processing as records, which includes creating a batch job instance in which processing takes place.
+//4
+. The batch job instance executes when the flow reaches `<process-records/>`. At this point, the xref:batch-phases.adoc#phase_process[Process phase] begins. All record processing takes place during this phase.
+//5
+. Each Batch Step component contains one or more processors that act upon a record to transform, route, enrich, or modify data in the records. For example, you might configure a connector operation to pass processed records one-by-one to an external server.
 +
-The queue name is prefixed with `BSQ`, and you can see it in Anypoint Runtime Manager:
+The only _unsupported_ processors are those that perform a request-response operation, such as an HTTP request, which returns a response. Mule processes a batch job as an asynchronous, one-way flow, so the results of batch processing do not return to the Mule flow, nor do the results return as a response to a caller.
+//6
+. A Batch Aggregator component is optional. You can add only one to a Batch Step component. The initial processor within a Batch Aggregator must be able to accept an array of records as input. Batch aggregation is useful for loading an array of processed records to an external server. It is also possible to use components, such as For Each, that iterate over the array so that other processors can process the records individually. The Batch Aggregator component requires a xref:batch-reference.adoc#ref_batch_aggregator_component[`streaming` or `size`] setting to indicate how to process records.
+//7
+. Additional Batch Step components are optional. This example does not contain a Batch Aggregator component.
+//8
+. After records pass through all Batch Step components, Mule completes the batch job instance and reports the results in an object that indicates which records succeed and which failed during processing. You can retrieve or log the result with a Logger or other processor within `<batch:on-complete />`.
 +
-image:mruntime-batch-bsq.png[Batch step queue transaction log]
+The Batch Job component consumes records in the xref:batch-phases.adoc#phase_on_complete[On Complete phase], which occurs after all record processing in a batch job instance finishes, and does not propagate the processed records to external components, outside of the Batch Job component. However, like many Mule components, the Batch Job component provides a way to set a xref:batch-reference.adoc#ref_batch_job_component[target variable] (`target`) property that a downstream component, outside Batch Job, can use to gain access to the _pre-processed payload_ that the Batch Job component receives.
```

Comment:
If you want to access the pre-processed payload you will not access the target variable but the payload (the target variable will have the result of the batch execution).

Reply:
... can use `payload` to gain access to the _pre-processed payload_ that the Batch Job component receives

Reply:
DONE: no change